sources/packages/backend/apps/queue-consumers/src/processors/schedulers/workflow/_tests_/assessment-workflow-enqueuer.scheduler.e2e-spec.ts

```diff
@@ -186,6 +195,78 @@ describe(
       expect(startApplicationAssessmentQueueMock.add).not.toBeCalled();
     });
 
+    it(`Should queue the assessment when an application has one assessment with status '${StudentAssessmentStatus.CancellationRequested}'.`, async () => {
```

Comment:
Maybe it should be clear about the queue as below.
"Should queue the assessment cancellation when an application has one assessment with status...".